docs/conf.py

```diff
@@ -109,109 +109,9 @@
-.. |TimeDelta| replace:: :class:`~astropy.time.TimeDelta`
-.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
-.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
-.. |Distribution| replace:: :class:`~astropy.uncertainty.Distribution`
-.. |Quantity| replace:: :class:`~astropy.units.Quantity`
-.. |Unit| replace:: :class:`~astropy.units.UnitBase`
-.. |Masked| replace:: :class:`~astropy.utils.masked.Masked`
-.. |PhysicalType| replace:: :class:`~astropy.units.PhysicalType`
-
-.. ------------------------------------------------------------------
-.. KNOWN PROJECTS
-
-.. _Python: https://www.python.org/
-.. |minimum_python_version| replace:: {__minimum_python_version__}
-
-.. Astropy
-.. _`Astropy mailing list`: https://mail.python.org/mailman/listinfo/astropy
-.. _`astropy-dev mailing list`: http://groups.google.com/group/astropy-dev
-
-.. NumPy
-.. _NumPy: https://numpy.org/
-.. _`numpy github`: https://github.com/numpy/numpy
-.. _`numpy mailing list`: http://mail.python.org/mailman/listinfo/numpy-discussion
-.. |minimum_numpy_version| replace:: {min_versions['numpy']}
-
-.. _numpydoc: https://pypi.org/project/numpydoc/
-
-.. erfa
-.. _ERFA: https://github.com/liberfa/erfa
-.. _PyErfa: http://pyerfa.readthedocs.org/
-.. _`pyerfa github`: https://github.com/liberfa/pyerfa/
-.. |minimum_pyerfa_version| replace:: {min_versions['pyerfa']}
-
-.. matplotlib
-.. _Matplotlib: https://matplotlib.org/
-.. |minimum_matplotlib_version| replace:: {min_versions['matplotlib']}
-
-.. sofa
-.. _SOFA: http://www.iausofa.org/index.html
-
-.. scipy
-.. _scipy: https://www.scipy.org/
-.. _`scipy github`: https://github.com/scipy/scipy
-.. _`scipy mailing list`: http://mail.python.org/mailman/listinfo/scipy-dev
-.. |minimum_scipy_version| replace:: {min_versions['scipy']}
-
-.. asdf
-.. |minimum_asdf_version| replace:: {min_versions['asdf']}
-
-.. pyyaml
-.. |minimum_pyyaml_version| replace:: {min_versions['pyyaml']}
-
-.. IPython
-.. _IPython: http://ipython.org/
-.. _`ipython github`: https://github.com/ipython/ipython
-.. _`ipython mailing list`: http://mail.python.org/mailman/listinfo/IPython-dev
-.. |minimum_ipython_version| replace:: {min_versions['ipython']}
-
-.. pip
-.. _pip: https://pip.pypa.io
-
-.. virtualenv
-.. _virtualenv: https://pypi.org/project/virtualenv
-.. _virtualenvwrapper: https://pypi.org/project/virtualenvwrapper
-.. _virtualenvwrapper-win: https://github.com/davidmarble/virtualenvwrapper-win
-.. _venv: https://docs.python.org/dev/library/venv.html
-
-.. conda
-.. _conda: https://conda.io/docs/
-
-.. py.test
-.. _pytest: https://pytest.org/en/latest/index.html
-.. _pytest-astropy: https://github.com/astropy/pytest-astropy
-.. _pytest-doctestplus: https://github.com/astropy/pytest-doctestplus
-.. _pytest-openfiles: https://github.com/astropy/pytest-openfiles
-.. _pytest-remotedata: https://github.com/astropy/pytest-remotedata
-
-"""
+with open("common_links.txt", "w") as cl:
```

Comment:
Docs failure is real, I think because:
```suggestion
with open("common_links.txt", "r") as cl:
```

Reply:
Lol. thanks! I opened this PR and moved to the next thing before ``tox -e build_docs`` has started up properly... so I think it tested my other branch (which passed TYVM)!

Reply:
Is there any way to speed up ``tox -e build_docs`` on the ``build_docs inst-nodeps: [path]/local/astropy/.tox/.tmp/package/1/astropy-4.3.dev1188+gf5883030e.tar.gz`` initial stage. It's sooo slow. Like cache some stuff?

Reply:
Oof, not that I know if, but if you figure out a way I would be keen to know! Related to #10470 I supposed..